app/styles/tokens/spacing.css

```
@@ -1,9 +1,9 @@
-
 :root {
-    --spacing: 0.25rem;
-    --spacing-sm: 0.5rem;
-    --spacing-md: 1rem;
-    --spacing-lg: 1.5rem;
-    --spacing-xl: 2rem;
-    --spacing-xxl: 4rem;
+    --spacing-sm: 0.25rem;
+    --spacing: 0.5rem;
+    --spacing-lg: 0.75rem;
+    --spacing-lg: 1rem;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--spacing-lg is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)